frontend/src/pages/MilestonesEditor.tsx

```diff
@@ -473,7 +474,7 @@ export const MilestonesEditor = () => {
                         {completed && <DoneAll />}
                         {version.name}
                       </div>
-                      {versionLists[version.id]?.length === 0 ? (
```

Comment:
This was the issue, comparing `undefined === 0` when the milestone was just added.